vaadin-confirm-dialog-flow-parent/vaadin-confirm-dialog-flow-integration-tests/src/test/java/com/vaadin/flow/component/confirmdialog/tests/helpers/Actionwords.java

```diff
@@ -1,98 +0,0 @@
-/*
- * Copyright 2000-2024 Vaadin Ltd.
- *
- * Licensed under the Apache License, Version 2.0 (the "License"); you may not
- * use this file except in compliance with the License. You may obtain a copy of
- * the License at
- *
- * http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
- * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
- * License for the specific language governing permissions and limitations under
- * the License.
- */
-package com.vaadin.flow.component.confirmdialog.tests.helpers;
-
-import java.util.Optional;
-
-import org.junit.Assert;
-
-import com.vaadin.flow.component.button.testbench.ButtonElement;
-import com.vaadin.flow.component.confirmdialog.ConfirmDialog;
-import com.vaadin.flow.component.confirmdialog.testbench.ConfirmDialogElement;
-import com.vaadin.flow.component.confirmdialog.tests.Features;
-import com.vaadin.testbench.ElementQuery;
-import com.vaadin.testbench.TestBenchElement;
-import com.vaadin.tests.AbstractParallelTest;
-
-public class Actionwords {
-
-    private final AbstractParallelTest test;
-    private final Class<?> testView;
-
-    public Actionwords(AbstractParallelTest test) {
-        this(test, Features.class);
-    }
-
-    public Actionwords(AbstractParallelTest test, Class<?> testView) {
-        this.test = test;
-        this.testView = testView;
-    }
-
-    Optional<ConfirmDialogElement> getConfirmDialog() {
-        ElementQuery<ConfirmDialogElement> query = test
-                .$(ConfirmDialogElement.class).onPage();
-        return query.exists() ? Optional.of(query.first()) : Optional.empty();
-    }
-
-    String getEventName() {
-        return test.$(TestBenchElement.class).id("eventName").getText();
-    }
-
-    public void iClickConfirm() {
-        getConfirmDialog().get().getConfirmButton().click();
-    }
-
-    public void confirmEventIsFired() {
-        checkFiredEventName(ConfirmDialog.ConfirmEvent.class);
-    }
-
-    public void iOpenDialogDialog(String dialog) {
-        test.$(ButtonElement.class).id(dialog).click();
-    }
-
-    public void iCompareTheDialogToReferenceImage(String dialog)
-            throws Exception {
-        test.compareScreen(dialog + "opened");
-    }
-
-    public void dialogDialogIsClosed(String dialog) {
-        Assert.assertFalse(getConfirmDialog().isPresent());
-    }
-
-    public void iHaveSampleDialog(String dialog) {
-        test.open(testView, AbstractParallelTest.WINDOW_SIZE_MEDIUM);
```

Comment:
I noticed that it's not used by the new tests, but it's fine since `AbstractComponentIT` sets the window size to 1024x800, which is almost the same as `WINDOW_SIZE_MEDIUM` = 1024x768.

Reply:
Yes, I also thought that resizing to `WINDOW_SIZE_MEDIUM` seems unnecessary so I removed it.